apps/admin-gui/src/app/admin/pages/admin-page/admin-services/service-detail-page/service-detail-page.component.ts

```diff
@@ -118,4 +125,28 @@ export class ServiceDetailPageComponent implements OnInit {
       () => (this.loading = false)
     );
   }
+
+  evaluateConsents() {
+    const config = getDefaultDialogConfig();
+    config.width = '500px';
+    config.data = {
+      title: this.translate.instant('SERVICE_DETAIL.CONFIRM_DIALOG_TITLE'),
+      theme: 'service-theme',
```

Comment:
```suggestion
      theme: 'service-theme',
      items: [this.service.name]
```
its weird when there is that line, I think it would be nice if there was the name of the service
![image](https://user-images.githubusercontent.com/32871167/166890573-e51d8e2d-1aa9-4aa2-bc6e-70a899b39307.png)